api/tests/conftest.py

```diff
@@ -0,0 +1,90 @@
+from os import environ, path
+
+import boto3
+import pytest
+import yaml
+from moto import mock_dynamodb2
+
+from hyp3_api import DYNAMODB_RESOURCE, auth, connexion_app  # noqa hyp3 must be imported here
+
+AUTH_COOKIE = 'asf-urs'
+JOBS_URI = '/jobs'
+
+DEFAULT_JOB_ID = 'myJobId'
+DEFAULT_USERNAME = 'test_username'
+
+
+@pytest.fixture
+def client():
+    with connexion_app.app.test_client() as test_client:
+        yield test_client
+
+
+@pytest.fixture()
+def table():
+    table_properties = get_table_properties_from_template()
+    with mock_dynamodb2():
+        boto3.setup_default_session()
+        table = DYNAMODB_RESOURCE.create_table(
+            TableName=environ['TABLE_NAME'],
+            **table_properties,
+        )
+        yield table
+
+
+def get_table_properties_from_template():
+    yaml.SafeLoader.add_multi_constructor('!', lambda loader, suffix, node: None)
+    template_file = path.join(path.dirname(__file__), '../../cloudformation.yml')
+    with open(template_file, 'r') as f:
+        template = yaml.safe_load(f)
+    table_properties = template['Resources']['JobsTable']['Properties']
+    return table_properties
+
+
+def make_job(granule='S1B_IW_SLC__1SDV_20200604T082207_20200604T082234_021881_029874_5E38',
+             description='someDescription',
+             job_type='RTC_GAMMA'):
+    job = {
+        'job_type': job_type,
+        'job_parameters': {
+            'granule': granule
+        }
+    }
+    if description is not None:
+        job['description'] = description
+    return job
+
+
+def make_db_record(job_id,
```

Comment:
only used by `test_list_jobs.py`, might move the definition there

Reply:
hrmm, I guess the quota check in `test_submit_jobs.py` needs it, so might as well leave it here